docs/modules/configuring/con-configuring-client-connections.adoc

```diff
@@ -82,4 +84,80 @@ If set to `REQUESTED`, the client is requested to present a certificate. If pres
 If set to `NONE`, client authentication is disabled.
 
 NOTE: The client's identity, as established through TLS client authentication, is currently not relayed to the target cluster. 
-For more information, see the https://github.com/kroxylicious/kroxylicious/issues/1637[related issue].
+For more information, see the {github-issues}/issues/1637[related issue].
+
+You may restrict the TLS protocols by either specifying an allow list of TLS protocols to be enabled, or
+specify a deny list of those protocols that should be disallowed. If you do not specify which protocols are to be used,
+the virtual cluster will use the defaults provided by the platform.
+
+When the client connects, the client and the virtual cluster will negotiate the highest mutually acceptable TLS protocol.
+If the two have no protocols in common, the connection will fail.
+
+The names of the TLS protocols are defined by {java-17-specs}/security/standard-names.html#sslcontext-algorithms[Java specification].
+
+.Example - restricting the TLS protocols using an allow list
+
+[source,yaml]
+----
+virtualClusters:
+  demo:
+    tls:
+      # ...
+      protocols:
+        allow:  # <1>
+        - TLSv1.3
+        - TLSv1.2
+----
+<1> List of TLS protocols to be allowed.
+
+.Example - restricting the TLS protocols using a deny list
+
+[source,yaml]
+----
+virtualClusters:
+  demo:
+    tls:
+      # ...
+      protocols:
+        deny:  # <1>
+        - TLSv1.1
+----
+<1> List of TLS protocols to be disallowed.
+
+You may restrict the TLS cipher suite by either specifying an allow list of cipher suite to be enabled, in preference
+order, or specify a deny list of those cipher suite that should be disallowed. If you do not specify which cipher suites
```

Comment:
I can't read this without wondering "what happens if I configure both?".

Reply:
If you say allow `A,B` and deny `B`, the result is `A`.   I don't expect a human to write that, but a computer might!